components/file_upload/file_upload.tsx

```diff
@@ -343,7 +332,7 @@ export class FileUpload extends PureComponent<Props, State> {
         const files: File[] = [];
         Array.from(droppedFiles).forEach((file, index) => {
             const item = items[index];
-            if (item && item.webkitGetAsEntry && (item.webkitGetAsEntry() === null || item.webkitGetAsEntry().isDirectory)) {
+            if (item && item.webkitGetAsEntry && (item.webkitGetAsEntry() === null || item.webkitGetAsEntry()?.isDirectory)) {
```

Comment:
we can simplify this a bit:

```suggestion
            if (item?.webkitGetAsEntry && (!item.webkitGetAsEntry() || item.webkitGetAsEntry()?.isDirectory)) {
```